.github/workflows/build.yml

```diff
@@ -7,28 +12,54 @@ on:
       - release-*
     tags:
       - v*
+    paths-ignore: 
+      - '**.md'
   pull_request:
     branches:
       - master
       - release-*
-
+    paths-ignore: 
+      - '**.md'
```

Comment:
this will block merge only if md files are changed. because GH checks will ensure all sample builds are done.

Reply:
OK. Changing it.